Core/include/Acts/Vertexing/AdaptiveMultiVertexFinder.hpp

```diff
@@ -189,11 +204,13 @@ class AdaptiveMultiVertexFinder {
   ///
   /// @param allTracks Input track collection
   /// @param vertexingOptions Vertexing options
+  /// @param state State for fulfilling interfaces
   ///
   /// @return Vector of all reconstructed vertices
   Result<std::vector<Vertex<InputTrack_t>>> find(
       const std::vector<const InputTrack_t*>& allTracks,
-      const VertexingOptions<InputTrack_t>& vertexingOptions) const;
+      const VertexingOptions<InputTrack_t>& vertexingOptions,
+      State& state) const;
```

Comment:
Is the empty `State` struct introduced for consistency ?

Reply:
Exactly. I will soon update the PR with information about everything that was changed/introduced. Basically all vertex finder now also get a ```State``` object, even though some don't need it and only have it for a consistent interface.